repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -112,21 +138,22 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
         Try(wrappedResponse.replaceHeader(
           CommonHttpHeader.LOCATION.toString,
           replaceIntoPath(originalLocation, token.get, previousToken, nextToken, None))) match {
-          case Failure(ex: PathRewriteException) => logger.warn("Failed while trying to rewrite the location header", ex)
           case Success(_) => //don't care
+          case Failure(e: PathRewriteException) => logger.warn("Failed while trying to rewrite the location header", e)
+          case Failure(e) => throw e
```

Comment:
I suppose keeping the original exception instead of accidentally wrapping it in a failure to match exception makes sense. This clearly thinking about the problems that were expected then.